index.html

```diff
@@ -3303,32 +3311,86 @@ <h3><dfn>Writing or pushing content</dfn></h3>
           </li>
           <li>
             If the type of a |record|'s <a>data</a> is not a
-            {{BufferSource}}, [= exception/throw =] a {{TypeError}}
-            and abort these steps.
+            {{BufferSource}} or {{NDEFMessageInit}}, [= exception/throw =] a
```

Comment:
As "sub records" are allowed for a local type record, we need to adjust https://pr-preview.s3.amazonaws.com/zolkis/web-nfc/pull/495.html#data-mapping as well.